bleak/backends/bluezdbus/scanner.py

```diff
@@ -141,53 +141,69 @@ async def start(self):
 
         logger.debug(f"cached devices: {self._cached_devices}")
 
-        # Apply the filters
+        await self.resume()
+
+    async def stop(self):
```

Comment:
why did you move it here? It makes the diff impossible to understand.

Reply:
Reorganised it now